internal/configs/version1/nginx-plus.tmpl

```diff
@@ -197,6 +199,7 @@ http {
     # NGINX Plus APIs
     server {
         listen {{.NginxStatusPort}};
+        listen [::]:{{.NginxStatusPort}};
```

Comment:
I think it makes sense to change the default value of https://github.com/nginxinc/kubernetes-ingress/blob/main/cmd/nginx-ingress/main.go#L134 to include the IPV6 localhost.  That value is used in this server in  the allow directive:
```
        {{range $value := .NginxStatusAllowCIDRs}}
        allow {{$value}};{{end}}
        deny all;
        location /api {
            api write=off;
        }
```

currently, the default value is `127.0.0.1`.

this endpoint is used  to access the dashboard https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/status-page/#accessing-live-activity-monitoring-dashboard 

same for NGINX OSS.